src/types/boolean.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongBoolean = StrongType<boolean>;
 
-export function makeBoolean(fallback: boolean, initial?: boolean | null): StrongBoolean {
+export function makeBoolean(fallback: boolean, initial: boolean | null | undefined): StrongBoolean {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.